packages/devtools_server/lib/src/chrome.dart

```diff
@@ -92,8 +91,8 @@ class Chrome {
         .firstWhere((line) => line.startsWith('DevTools listening'));
 
     return _connect(Chrome._(
```

Comment:
this doesn't make sense without the --remoteDebuggingPort option. Remove this code. Someone who wants to connect to the debug port can do that separately in a wrapper method.

Reply:
actually never mind. we can keep this but we should handle a null debugPort better.

Reply:
removed parts of this code we don't need. I will add them back when I create the browser_launcher package.